### ISSUE TITLE
Show the Amount of SNX being staked before confirming the transaction

### ISSUE THREAD

**Issue by hav-noms** [Synthetixio/synthetix-mintr#91]

**Comment by marknl** [comment]
I'd love to have the option of setting a custom c-ratio.

**Comment by crypto-dad** [comment]
Hell yea. Would love to have this and nothing to add--it covers all 3 items on my wishlist. Currently I input my desired sUSD to mint, click Mint Now, and check the screen to see how much SNX will be staked so I can maintain a buffer of SNX unstaked to uphold my c-ratio.

**Comment by clementbalestrat** [comment]
@hav-noms what other info should we add? C-ratio?

**Comment by hav-noms** [comment]
Yes please. Add c-ratio label with what the users new "estimated" c-ratio will be. Of course with price movements none of these preview values are guaranteed. Probably a *note we want to display on the UI also.
